lib/matplotlib/tests/test_offsetbox.py

```diff
@@ -378,3 +378,13 @@ def test_packers(align):
         x_height = (x2 - x1) / 2
     # x-offsets, y-offsets
     assert_allclose([(x_height, 0), (0, -y2)], offset_pairs)
+
+
+def test_paddedbox():
+    fig, ax = plt.subplots()
+
+    at = AnchoredText("foo",  'upper left')
+    try:
+        pb = PaddedBox(at, patch_attrs={'facecolor': 'r'}, draw_frame=True)
+    except:
+        raise Exception("incorrect default value")
```

Comment:
There is no need to do this in a test. The test will fail if there is an exception anyway, so skip the try-except.

You can add a comment like

```
# smoke test for correct default value though
```